packages/react-instantsearch-hooks/README.md

```diff
@@ -468,6 +468,133 @@ function Hits(props) {
 }
 ```
 
+
+### `useMenu`
+
+> `(props: UseMenuProps) => MenuRenderState`
+
+Hook to use a [menu](https://www.algolia.com/doc/api-reference/widgets/menu/js/).
+
+**Types**
+
+<details>
+<summary><code>UseMenuProps</code></summary>
+
+```ts
+type UseMenuProps = {
+  /**
+   * The name of the attribute in the records.
+   */
+  attribute: string;
+  /**
+   * The max number of items to display when
+   * `showMoreLimit` is not set or if the widget is showing less value.
+   */
+  limit?: number;
+  /**
+   * Whether to display a button that expands the number of items.
+   */
+  showMore?: boolean;
+  /**
+   * The max number of items to display if the widget
+   * is showing more items.
+   */
+  showMoreLimit?: number;
+  /**
+   * How to sort refinements. Possible values: `count|isRefined|name:asc|name:desc`.
```

Comment:
I'm not sure it's designed for values, but could we leverage the [`@enum`](https://jsdoc.app/tags-enum.html) directive here?

Reply:
typescript is already sufficient for that no?

Reply:
I was mentioning it because you documented it in the JSDoc although it's already in the types. But after checking closer, `@enum` isn't really designed for that.